cmd/ledgerutil/main.go

```diff
@@ -53,21 +53,22 @@ func main() {
 			*outputDir, err = os.Getwd()
 			if err != nil {
 				fmt.Printf("%s%s\n", compareErrorMessage, err)
-				return
+				os.Exit(1)
 			}
 		}
 
 		count, outputDirPath, err := ledgerutil.Compare(*snapshotPath1, *snapshotPath2, *outputDir, *firstDiffs)
 		if err != nil {
 			fmt.Printf("%s%s\n", compareErrorMessage, err)
-			return
+			os.Exit(1)
 		}
 
 		fmt.Print("\nSuccessfully compared snapshots. ")
 		if count == -1 {
 			fmt.Println("Both snapshot public state and private state hashes were the same. No results were generated.")
 		} else {
 			fmt.Printf("Results saved to %s. Total differences found: %d\n", outputDirPath, count)
+			os.Exit(1)
```

Comment:
This is not an error scenario for the tool. The tool is meant to find the differences.

Reply:
@manish-sethi Thank you for your review. While I agree that this is not an error case, I think it is more convenient for users to know if there is any difference by the exit code as they can know with diff(1). Does returning a different code, say 2, make sense?

Reply:
Sure, not a big deal if you want it to be distinguishable outcome. In that case, yes, better to use a different exit code.
If you want to go ahead with this change, please update the documentation as well.

BTW, do you have a scenario in mind where you would want to include this util as a part of some larger workflow where you would like to decide on the next step based on whether the snapshot are same or different?

Reply:
Also, in that case, I would suggest make this exit code as `0` and the one above it (`if count == -1`) as `1` as the one above is an exit point because of wrong input and this one is a default intended exit point.